src/meltano/core/bundle/settings.yml

```diff
@@ -12,7 +12,7 @@ settings:
   # Replaces negated env_alias from `send_anonymous_usage_stats`
 - name: project_id
 - name: database_uri
-  value: sqlite:///$MELTANO_PROJECT_ROOT/.meltano/meltano.db
+  value: sqlite:///$MELTANO_SYS_DIR_ROOT/meltano.db
```

Comment:
Can someone confirm this works in manual testing? 

If so, would lean towards disproving the issue described here:

- #6920

Reply:
@aaronsteers Yeah, I can confirm that the interpolation works fine in this use case:

```console
$ export MELTANO_SYS_DIR_ROOT=$HOME/.meltano-data

$ meltano init new-project
...

$ ls -a | grep "^\."

.
..
.gitignore

$ tree $MELTANO_SYS_DIR_ROOT
/Users/edgarramirez/.meltano-data
├── analytics.json
├── meltano.db
├── meltano.db-shm
├── meltano.db-wal
└── run
    ├── bin -> /Users/edgarramirez/Library/Caches/pypoetry/virtualenvs/meltano-SY7IjDqw-py3.10/bin/meltano
    └── meltano.yml.lock

$ meltano config meltano list | grep "DATABASE_URI"
database_uri [env: MELTANO_DATABASE_URI] current value: 'sqlite:////Users/edgarramirez/.meltano-data/meltano.db' (default: 'sqlite:///$MELTANO_SYS_DIR_ROOT/meltano.db')
```

Reply:
@aaronsteers Is there any other check that comes to mind?

Reply:
@edgarrmondragon - No, that's great. Thanks for confirming. 👍